sdk/Lusid.Sdk/Lusid.Sdk.csproj

```diff
@@ -1,8 +1,8 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
-    <Version>0.11.3040</Version>
+    <Version>0.11.3028-MARK</Version>
+    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
Note that  ".NET Framework doesn't support .NET Standard 2.1. For more information, see the announcement of .NET Standard 2.1" - https://docs.microsoft.com/en-us/dotnet/standard/net-standard

According to this document - "We recommend you target the lowest version of .NET Standard possible. So, after you find the highest .NET Standard version you can target, follow these steps..." It then outlines the steps to find the lowest version possible.

As we are not doing anything fancy, let's follow these steps and find the lowest version of netstandard that we can support.

Reply:
oh that's a cool page! I found some other pages but didn't realise that .NET Framework doesn't support .NET Standard 2.1. So then we may as well go back to .NET Standard 2.0

Reply:
Okay, let's make this 2.0 then!

Reply:
Or can we go even lower to add even more support?

Reply:
I tried to go lower, but then we can't use RestSharp:

> <PackageReference Include="RestSharp" Version="106.11.7" />

I guess maybe that only became available in .NET Standard 2.0

Reply:
Seems like it's intentionally only for 2.0 and above - https://github.com/restsharp/RestSharp/issues/992

Reply:
Perfecto!